tracer/build/_build/Build.Steps.cs

```diff
@@ -1053,6 +1040,74 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                 .SetMaxCpuCount(null));
         });
 
+    Target CompileDebuggerIntegrationTests => _ => _
+        .Unlisted()
+        .After(CompileManagedSrc)
+        .DependsOn(CompileDebuggerIntegrationTestsDependencies)
+        .DependsOn(CompileDebuggerIntegrationTestsSamples)
```

Comment:
Just thinking about local testing - do you _always_ want to have to rebuild the dependencies + samples when building the debugger integration tests? If no, make these `After()`, and add additional `DependsOn()` calls to the "BuildAndRunDebuggerIntegrationTests` target. Entirely up to you 🙂

Reply:
The only one who uses whose things locally is me.
And I **always** want to recompile those things. 😁